src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/entities/Direction.java

```diff
@@ -46,6 +46,7 @@ public Direction(
         requireNonNull(dateSent);
         requireNonNull(tag);
         requireNonNull(previousDates);
+        requireNonNull(clarifyingQuestions);
```

Comment:
I think clarifying questions can be null. Need to be checked. The two cases I can think of 
1) are if we do a normal direction without questions (think we set clarifying questions to an empty list in this case but not sure). 
2) Backward compatibility when we are working with a direction that was created before we added clarifying questions. Think in this case they will be null. Expect this constructor may not get called then but needs to be checked.